lnwire/features.go

```diff
@@ -161,6 +161,26 @@ const (
 	// sender-generated preimages according to BOLT XX.
 	AMPOptional FeatureBit = 31
 
+	// WantStorageRequired is a required feature bit that indicates
+	// that a node wants to store their data with other peers.
+	// See: https://github.com/lightning/bolts/pull/1110/
+	WantStorageRequired FeatureBit = 40
+
+	// WantStorageOptional is an optional feature bit that indicates
+	// that a node wants to store their data with other peers.
+	// See: https://github.com/lightning/bolts/pull/1110/
+	WantStorageOptional FeatureBit = 41
```

Comment:
I think the `option_want_storage` feature bit might be removed from the spec. Currently, there are no requirements of the storage provider to check this feature bit at all afaict. Since this PR is also only focusing on the provider, I dont think we need to add these here. 

see threads [here](https://github.com/lightning/bolts/pull/1110#discussion_r1519349651) and [here](https://github.com/lightning/bolts/pull/1110#discussion_r1539503055).